buildbot.mariadb.org/constants.py

```diff
@@ -6,9 +11,17 @@
 
 builders_big=["amd64-ubuntu-1804-bigtest"]
 
-builders_install=["amd64-debian-11-deb-autobake-install", "amd64-ubuntu-1804-deb-autobake-install", "amd64-centos-7-rpm-autobake-install", "amd64-fedora-35-rpm-autobake-install"]
-
-builders_upgrade=["amd64-ubuntu-1804-deb-autobake-major-upgrade", "amd64-ubuntu-1804-deb-autobake-minor-upgrade", "amd64-centos-7-rpm-autobake-major-upgrade", "amd64-centos-7-rpm-autobake-minor-upgrade"]
+# Generate install builders based on the os_info data
+builders_install = []
+builders_upgrade = []
+for os in os_info:
+    for arch in os_info[os]['arch']:
+        builder_name_install = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake-install'
```

Comment:
can/should we assert that the non-autobake builder is there saving packages?

Reply:
For now I don't have a good idea. The goal is to generate everything based on the .yaml file so a check would not be needed